src/Chainweb/Miner/RestAPI/Server.hs

```diff
@@ -60,18 +62,29 @@ import Data.Singletons
 
 ---
 
+-- | KILLSWITCH: The logic here involing `txSilenceDates` is to be removed in a
+-- future version of Chainweb. This logic errors on remote requests for new
+-- mining work, such that this Node can no longer meaningfully participate in
+-- mining.
+--
```

Comment:
What about using a similar approach as with the mempool, of just completely disabling the miner API? What that be easier to implement?